agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -93,14 +93,26 @@ while true; do
 		;;
 	esac
 done
+
+function setup_ara {
+	generate_inventory >> /tmp/inventory.$$
+	check_install_rpm pbench-ara-role
+	ansible-playbook -i ${INVENTORY} /opt/pbench-agent/ansible/ara/pbench-ara.yml &>$pbench_log
+}
 	
 if $check; then
-	if [ "$sysinfo" == "all" ] || [ "$sysinfo" == "default" ] || [ "$sysinfo" == "none" ]; then
+	if [ "$sysinfo" == "default" ] || [ "$sysinfo" == "none" ]; then
 		:
+	elif [ "$sysinfo" == "all" ]; then
+		# generate inventory with pbench controller and remotes, install and configure ara
+		setup_ara
```

Comment:
ara needs to be setup and started before a benchmark run starts, so that we can collect the data once the benchmark run completes. So when the sysinfo option is all, we need to setup ara as well.